site/docs/hosting/vps.md

```diff
@@ -7,24 +7,125 @@ next: false
 
 A virtual private server, mostly known as VPS, is a virtual machine running in the cloud with its users having the full control of its system.
 
-In this guide, you'll learn about various methods of running your bot in a VPS, keeping it online 24/7, making it run automatically when your VPS boots and restart on crashes.
+## Server Rental
+
+> You need to rent a VPS in order to follow this guide.
+> This first section will explain how to do that.
+> If you already have a VPS to work on, skip [down](#starting-the-bot).
+
+We choose [Hostinger](https://hostinger.com) as our host.
+
+> You are free to choose the provider of your choice.
+> All providers provide the same services, so you won't have any problems with the technical part of this article.
+> You can take this part as an overview of how the server rental operation works.
+> If you are a beginner, you can use this guide to rent your first ever server!
+
+::: tip Analog of a server
+If you cannot or do not want to rent a server but you still want to play around with running a bot on a VPS, you can follow this tutorial on a virtual machine instead.
+To do this, use an application such as [VirtualBox](https://virtualbox.org).
+Create a virtual machine with the desired Linux distribution and use it as a real server.
+:::
+
+Go to the [VPS tariff catalog](https://hostinger.com/vps-hosting).
+We will use the "VPS 1" plan.
+Below, you can see a comparison table that describes the offered server resources and limits.
+The resources of VPS 1 are enough for bots with a small audience, and even more so for our test bot.
+
+Go back to the "VPS 1" card and click the "Add" button.
+You will be automatically redirected to the checkout page, where you will also immediately register with Hostinger.
+
+::: warning Change the rental term!
+The typical lease term is 2 years (a marketing ploy), and it costs a lot of money.
+You probably don't need it, so you can start by renting a server for a month, which is much cheaper.
+
+In any case, Hostinger provides a 30-day money-back guarantee.
+:::
+
+After payment, you will be able to set up your server:
+
+1. **Location.**
+   We recommend that you choose the location closest to Amsterdam.
+   It is in Amsterdam that the Telegram server is located, which is responsible for ensuring the operation of bots.
+   If you use your own Bot API server, choose the location closest to it.
+2. **Server type.**
+   Choose the option "Clean OS".
```

Comment:
```suggestion
   Choose the option "Clean OS."
```
is technically the right way to do it in English, even though I don't like this. @roj1512?

Reply:
I think like that, too.

![image](https://github.com/grammyjs/website/assets/49933115/8fe06a3b-7327-4615-8dec-f5a474f02069)

Reply:
Apple is using `""`? Blasphemy.

Reply:
Yes, I also thought that. Perhaps they were lazy.